modules/serverless-create-default-channel.adoc

```diff
@@ -0,0 +1,50 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-channels.adoc
+
+[id="serverless-create-default-channel_{context}"]
+= Creating a channel with cluster or namespace defaults
```

Comment:
Should this include "or namespace defaults" ? currently it only includes the cluster default